### ISSUE TITLE
Crop image causes crashes

### ISSUE THREAD

**Issue by tetzlav** [hyperion-project/hyperion.ng#1445]
- [x] I confirm that this is an issue rather than a question.

## Bug report

I installed Hyperion.NG 2.0.12 in Libreelec 9.2.8 on a Raspberry 3B with Paulchen-Panther's script. I would like to use a Mirabox HSV321 as a grabber.

The whole configuration worked great. I saw in the live image that there were still black edges all around.

So I wanted to crop the edges. But as soon as I crop and save a pixel number greater than 0 somewhere on the edge, hyperiond crashes and gets stuck in a restart loop. This can be reproduced and can only be remedied with the --deleteDatabase parameter.

#### Log
```
2022-03-09T20:43:05.669  V4L2                  : <INFO> Cropping image: width=640 height=480; crop: left=1 right=0 top=0 bottom=0
2022-03-09T20:43:05.671  V4L2                  : <INFO> Reloading V4L2 Grabber
2022-03-09T20:43:05.672  V4L2                  : <DEBUG> V4L2Grabber.cpp:122:uninit() | Uninit grabber: MiraBox Video Capture: MiraBox (/dev/video0)
2022-03-09T20:43:05.675  V4L2                  : <INFO> Stopped
2022-03-09T20:43:05.680  V4L2                  : <DEBUG> V4L2Grabber.cpp:151:init() | Set device (path) to: MiraBox Video Capture: MiraBox (/dev/video0)
2022-03-09T20:43:05.681  V4L2                  : <DEBUG> V4L2Grabber.cpp:488:init_device() | Set device input to: Camera 1
2022-03-09T20:43:05.695  V4L2                  : <DEBUG> V4L2Grabber.cpp:614:init_device() | Set resolution to width=640 height=480
2022-03-09T20:43:05.707  V4L2                  : <DEBUG> V4L2Grabber.cpp:631:init_device() | Set framerate to 30 fps
2022-03-09T20:43:05.709  V4L2                  : <DEBUG> V4L2Grabber.cpp:739:init_device() | Pixel format=MJPEG
2022-03-09T20:43:05.718  V4L2                  : <INFO> Started
free(): invalid pointer

Hyperion caught signal :SIGABRT
2022-03-09T20:43:05.909  CORE                  : <ERROR>        /opt/vc/lib/libc.so.6(__default_rt_sa_restorer+0) [0x756a8210]
Aborted
```

#### System
```
Hyperion Server: 
- Build:           (HEAD detached at df149583) (GitHub-dc6aa4dd/df149583-1637501177)
- Build time:      Nov 21 2021 17:01:51
- Git Remote:      https://github.com/hyperion-project/hyperion.ng
- Version:         2.0.12
- UI Lang:         de (BrowserLang: de)
- UI Access:       expert
- Avail Capt:      dispmanx,v4l2,framebuffer,qt
- Config path:     /storage/hyperion
- Database:        read/write

Hyperion Server OS: 
- Distribution:   LibreELEC (official): 9.2.8
- Architecture:   arm
- CPU Model:      ARMv7 Processor rev 4 (v7l)
- CPU Type:       Raspberry Pi 3 Model B Plus Rev 1.3
- CPU Revision:   a020d3
- CPU Hardware:   BCM2835
- Kernel:         linux (4.19.127 (WS: 32))
- Root/Admin:     true
- Qt Version:     5.7.1
- Python Version: 3.5.3
- Browser:        Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:97.0) Gecko/20100101 Firefox/97.0
```

**Comment by Paulchen-Panther** [comment]
The bug should be fixed already (#1399). Unfortunately, we don't have a date for a new release yet. To provide you with an alternative, download a recent PR artifact (armv6l) ([link](https://github.com/hyperion-project/hyperion.ng/actions/runs/1988972622)) and replace the hyperiond file from the unpacked artifact on your LibreELEC under /storage/hyperion/bin.

**Comment by tetzlav** [comment]
yes, is fixed in 2.0.13-beta.1+PR1444
